### DISCUSSION TITLE
吃什么菜单介意加餐吗

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#642] [General]
群友给我整的，我大为震撼
<img width="1082" alt="image" src="https://user-images.githubusercontent.com/9006264/153024127-28449745-c4ae-483f-a276-0cf23be77b8d.png">
![image](https://user-images.githubusercontent.com/9006264/153024135-0d8ffe06-8ac4-4d5a-9721-afad141c8b0d.png)

**Comment by Arondight** [comment]
不介意，随便改，现在有的我都是随便百度了个菜单粘贴复制的

**Reply by Arondight** [reply]
但是不要加不好的东西，曾经有人给我在菜单里提交了一个💩进去

**Reply by Arondight** [reply]
你要是愿意规整一下更好，现在都是乱写的，总之任意增删

**Reply by mark9804** [reply]
> 但是不要加不好的东西，曾经有人给我在菜单里提交了一个💩进去

那个提交我之前看到过，我自己是干不出这种事…

**Comment by mark9804** [comment]
因为加上之后项目有点多，我觉得可以把一些三餐都能吃的东西合并起来，早午晚餐字段只维护各自的内容，最后concat起来

然后想确认一下，现在获取随机项目范围是array.length，不用再 -1 了对吧

**Reply by Arondight** [reply]
> 因为加上之后项目有点多，我觉得可以把一些三餐都能吃的东西合并起来，早午晚餐字段只维护各自的内容，最后concat起来
> 
> 然后想确认一下，现在获取随机项目范围是array.length，不用再 -1 了对吧

我感觉没必要这么麻烦，简单些就好



> 然后想确认一下，现在获取随机项目范围是array.length，不用再 -1 了对吧

以前也是这样，从未用过 length - 1

**Comment by mark9804** [comment]
老大哥，我试着改了一下 265057e67caff4d1b271bb565d0b4afd514a7e68 ，但是很尴尬不知道为什么 select case 总是落到 default ，有空的话能不能帮忙看一下

https://github.com/Arondight/Adachi-BOT/blob/265057e67caff4d1b271bb565d0b4afd514a7e68/src/plugins/tools/menu.js#L46-L56

**Reply by Arondight** [reply]
额，喝的都加到三餐里就行了，这个功能本来就跟抽奖的差不多，并没有人在 bot 的菜单是什么，只要别出恶心人的东西就行

**Reply by Arondight** [reply]
我看你代码都写完了，我上班给你改一下，配置文件我也要改一下

**Reply by Arondight** [reply]
做完核酸了，回去我开电脑瞅瞅

**Reply by Arondight** [reply]
心态炸裂，提交之前 checkout 了
